### PR TITLE
feat: descriptive open full page action

### PR DESCRIPTION
**Pull request by kyranjamie**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/blockstack/ux/actions/runs/1121005181).

"Options" doesn't provide a clear meaning of it's action: opening wallet in new page.

This PR adds a descriptive context menu item.

image

cc/ @aulneau @kyranjamie @fbwoolf

### CODE REVIEW

**Review by beguene — APPROVED** [review]
Finally ! 😄

File: src/background/init-context-menus.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+function openNewTabWithWallet() {
+  return chrome.tabs.create({ url: 'full-page.html' });
+}
+
+export function initContextMenuActions() {
+  chrome.contextMenus.removeAll();
+
+  chrome.contextMenus.create({
+    title: 'Open Hiro Wallet full page',
```
Comment (aulneau) [inline src/background/init-context-menus.ts]: I wonder if this should be something else, like `Open in full page mode`. cc @jasperjansz for ideas

**Review by aulneau — APPROVED** [review]

File: src/background/init-context-menus.ts (hunk 1 above)
Comment (kyranjamie) [reply src/background/init-context-menus.ts]: Yep down for phrasing alternatives cc/ Mark often-has-opinions-on-copy Hendrickson @markmhx

Comment (kyranjamie) [reply src/background/init-context-menus.ts]: ...in a new tab might also be better terminology

**Review by fbwoolf — APPROVED** [review]

File: src/background/init-context-menus.ts (hunk 1 above)
Comment (aulneau) [reply src/background/init-context-menus.ts]: @kyranjamie how about `Open Hiro wallet in new tab` and merge it?

**Review by aulneau — APPROVED** [review]

**Pull request merged**